pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -1146,3 +1150,112 @@ def sample_posterior_predictive(
             )
 
         return posterior_predictive_samples
+
+    def add_lift_test_measurements(
+        self,
+        df_lift_test: pd.DataFrame,
+        dist: pm.Distribution = pm.Gamma,
+        name: str = "lift_measurements",
+    ) -> None:
+        """Add lift tests to the model.
+
+        The empirical difference of a channel's saturation curve is created
```

Comment:
Total nitpicker here, but I think you're using the term "empirical" wrong starting here. What you're actually doing below (line 277) is estimating the "theoretical" or "model estimated" lift. The *empirical* lift if actually `delta_y`.

Reply:
Thanks. Good feedback. 

I will go with the distinction
1. that the model evaluated difference will be "model_estimated"
2. delta_y = "empirical_lift"